AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRColorGradingAdded.py

```diff
@@ -146,42 +304,427 @@ def AtomEditorComponents_HDRColorGrading_AddedToEntity():
+                          AtomComponentProperties.hdr_color_grading('White Balance Weight')) == 1.0)
+
+        # 17. Temperature (float range 1000.0 to 40000.0)
+        # Set Temperature to its minimum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Temperature'), 1000.0)
+        Report.result(Tests.temperature_min_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Temperature')) == 1000.0)
+
+        # Set Temperature to its maximum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Temperature'), 40000.0)
+        Report.result(Tests.temperature_max_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Temperature')) == 40000.0)
+
+        # 18. Tint (float range -100.0 to 100.0)
+        # Set Tint to its minimum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Tint'), -100.0)
+        Report.result(Tests.tint_min_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Tint')) == -100.0)
+
+        # Set Tint to its maximum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Tint'), 100.0)
+        Report.result(Tests.tint_max_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Tint')) == 100.0)
+
+        # 19. Luminance Preservation (float range -1.0 to 1.0)
+        # Set Luminance Preservation to its minimum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Luminance Preservation'), -1.0)
+        Report.result(Tests.luminance_preservation_min_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Luminance Preservation')) == -1.0)
+
+        # Set Luminance Preservation to its maximum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Luminance Preservation'), 1.0)
+        Report.result(Tests.luminance_preservation_max_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Luminance Preservation')) == 1.0)
+
+        # 20. Split Toning|Weight (float range 0.0 to 1.0, default 0.0)
```

Comment:
do we know that this value remains default 0.0? you skipped setting this back to 0.0; what if the minimum changes to 0.5. I doubt that would happen but this might not catch that.

Reply:
The Weights in all of these parameters are effectively enabling/disabling that series of sub-parameters. 0.0 being disabled, 1.0 being fully enabled, and everything in between making it have less overall effect on the scene.

There wouldn't be a reason in the future to change the minimum values on this (that's why I decided to omit the check to set to minimum)